tiledb/sm/query/readers/sparse_index_reader_base.cc

```diff
@@ -240,8 +240,17 @@ std::vector<uint64_t> SparseIndexReaderBase::tile_offset_sizes() {
           num += deletes_consolidation_no_purge_;
         }
 
+        // Other than the offsets themselves, there is also memory used for the
+        // initialization of the vectors that hold them. This initialization
+        // takes place in fragment_metadata.cc::load_footer()
+        unsigned num_vectors = schema->attribute_num() + 1 +
```

Comment:
Document the +1. I think it's the old zipped coords?

Reply:
This is the only thing I don't understand. What this +1 stands for. It is documented nowhere